start_pgbouncer.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+PGB_DIR=/etc/pgbouncer
+INI=$PGB_DIR/pgbouncer.ini
+USERLIST=$PGB_DIR/userlist.txt
```

Comment:
```suggestion
PGB_DIR="/etc/pgbouncer"
INI="${PGB_DIR}/pgbouncer.ini"
USERLIST="${PGB_DIR}/userlist.txt"
```

Please get in the habit of quoting shell script variables :)